examples/src/main/res/raw/building_extrusion_highlight_activity.txt

```diff
@@ -0,0 +1 @@
tion":[-122.414021,37.790917],"bearing_before":171.0,"bearing_after":0.0,"instruction":"You have arrived at your destination","type":"arrive"},"voiceInstructions":[],"bannerInstructions":[],"driving_side":"right","weight":0.0,"intersections":[{"location":[-122.414021,37.790917],"bearings":[351],"entry":[true],"in":0}]}],"annotation":{"distance":[4.377558046781132,29.048645741202307,19.932752580690952,48.576014751952926,11.943930596414795,12.720147968330114,34.58112560559672,50.96480933738249,10.349556668565532,10.428744124966986,26.062939337835754,33.61207745353073,22.152629772397148,11.584122498337331,8.22504067141732,73.25791925952063,11.603839142628226,1.7930152150745173,1.9339724019326627,6.4182775391464615,8.137686480966448,3.8827293592940286,4.137411880032337,15.778927074065876,36.18605026816596,7.882482295429149,9.70628239052936,44.88520535070527,19.777529614845466,18.70069668033605,2.7608352091952852,8.820943994209998,9.167951891088123,20.211271187411434,67.05841934336348,8.004641462478803,9.602196312248573,50.45852431578543,12.710488322858675,9.917381766860117,27.67756036992424,21.374879966188537,14.151341864692084,11.581827207587462,103.10585401718104,20.975906679342675,9.863007930072529,27.750876716852083,31.428390784757813,50.12927675688965,26.70541846540595,124.91524890087112,9.619673955896003,7.744241825057303,9.081282300519934,55.824774196397755,72.99751351106848,4.01125146327413,10.354407638921565,86.47389715542175,9.46174014762768,1.579336360053304,1.456588625710653,11.030579241536637,43.68327188987056,25.693275300415763,16.824625284570534,5.792905661239118,2.381194445403029,1.337050795424019,9.359356548680744,20.074574908738008,117.23790846433582,76.81587623559835,3.7514168675917774,27.764134475011225,37.66153518871731,136.19947456061075,9.984630741768928,91.22535667168732,13.293150952776184,1.90914496306627,8.555399787319097,2.4861312264216893,43.69699404532718],"congestion":["moderate","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","moderate","low","low","low","low","low","moderate","moderate","low","low","moderate","moderate","moderate","moderate","low","low","low","low","low","low","low","moderate","moderate","low","low","low","low","low","low","moderate","low","low","low","low","low","low","low","low","low","low","moderate","low","low","low","low","low","low","low","low","low","low","low"]}}],"routeOptions":{"baseUrl":"https://api.mapbox.com","user":"mapbox","profile":"driving-traffic","coordinates":[[-122.396796,37.7918406],[-122.4140199,37.7909168]],"alternatives":true,"language":"en","continue_straight":false,"roundabout_exits":false,"geometries":"polyline6","overview":"full","steps":true,"annotations":"congestion,distance","voice_instructions":true,"banner_instructions":true,"voice_units":"metric","access_token":"pk.ey","uuid":"ckblcybpcnz7379uen64vrcuj"},"voiceLocale":"en-US"}
```

Comment:
We were using same route string in different examples. We should check which and remove the duplicated ones. Same applies to rest of the routes 👇

Reply:
fixed